### PR TITLE
Typography 2016 10 18

### PR DESCRIPTION
**Pull request by davidpmccormick**

### CODE REVIEW

**Comment by jamesgorrie** [comment]
Any chance of a description as to why you did this?

File: server/views/patterns/typography.njk [hunk 1]
```diff
@@ -123,257 +121,254 @@ p q r s t u v w x y z { | } ~
   </div>
 
   <h3>Figures</h3>
-  <p>Figures are usually used to refer to images:</p>
+
+  <h4>Image-based figure</h4>
   <div class="example">
     <figure>
-      <img src="http://lorempixel.com/680/400/abstract" alt="Example image" />
+      <img style="max-width: 100%;"
```
Comment (jamesgorrie) [inline server/views/patterns/typography.njk]: We could start putting these into components / functions with arguments and validations?

Comment (davidpmccormick) [inline server/views/patterns/typography.njk]: Preach!

But I think we need more than *just* an e.g. image-with-caption to start building components – we're essentially treading water a bit until we get something a bit more sured-up out of C&T. Which will hopefully be any day soon.

This also largely answers the 'why' of this PR – essentially just tidying up/removing cruft for want of something more substantial to be getting our teeth in to.

Comment (jamesgorrie) [inline server/views/patterns/typography.njk]: Image with caption, I image would have an image component in it no? To stop having to write this out leaving loads of room for forgetting something.

Nice.

**Pull request merged**